lib/rails_db.rb

```diff
@@ -17,6 +17,10 @@ module RailsDb
   mattr_accessor :enabled
   @@enabled = true
 
+  # Allow read only operations
+  mattr_accessor :sandbox
+  @@sandbox = true
```

Comment:
the default behavior for sandbox should be `true`?

Reply:
No, you are right.
I will change to false.

Thanks